part-4/model-building-components.md

```diff
@@ -564,15 +566,111 @@ The screen should look like in the screen shot below:
 
 ![Container Observer for Sum of Metabolites](../assets/images/part-4/ContainerObserverEntered.png)
 
-## Events and Applications‌
+# Events Building Block
+
+An **event** is used to change an entity, like the amount of molecules or a reaction rate, when a given condition is met. This condition can be, for example, that a given simulation time is reached, or that the concentration of a molecule has exceeded a certain value. Thus, such a programmed event is used to reflect external changes to the simulation, like the application of a drug or a sudden physical change in the spatial structure, like a vessel rupture.
+
+A special case of an event is an **application**, which is used to apply a certain amount of a molecule to a container at a given time. This is typically used to simulate the administration of a drug to the body.
+
+
+
+Events and applications are grouped in the **Events** building block (BB). Multiple events can be grouped in event groups, and organized in a hierarchy of  containers similar to that of a [spatial structure](spatial-structures-bb.md). Simulations imported from a PK-Sim® project contain all defined applications and additional events, such as food administration, in the events BB.
+
+The following section describes the functionalities of the Events building block based on a PBPK model exporter from PK-Sim. Later on, a simple [example](#example---creating-events) is given. Since the generation of an application in MoBi® can be rather complicated and is beyond the scope of this manual, we will restrict the description to adapting applications that were previously imported from PK-Sim®, where complex applications schemes can be generated more easily.
+
+## Events - Functionality Overview‌
+
+Events are organized in a tree structure. The top level container of the Events BB can be either an **Event Group** or an **Application**.‌
+
+### Event Group
+
+Event groups can combine **Applications**, **Events**, further **Event Groups**, and **Containers**.‌
+
+An event group can be created within:
+    - another event group,
+    - Application
+
+The top level event group has a *name*, the *container criteria* which determines where the events group will be created, and a set of *parameters*.
```

Comment:
_... which determines where the events group will be created **in a simulation,**..._

Reply:
added